### PR TITLE
update the connection button at every connection change

### PR DESCRIPTION
**Pull request by gdorsi**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -247,23 +247,28 @@ function refreshMyApps() {
...
     refreshLibrary();
   }).catch(err => {
     showToast("Getting app list failed, "+err,"error");
   });
 }
 
+var connectMyDeviceBtn = document.getElementById("connectmydevice");
+
+function handleConnectionChange(connected) {
+  connectMyDeviceBtn.style.display = connected ? 'none' : '';
```
Comment (gdorsi) [inline index.js]: Would make sense to clean the library on disconnection?

**Comment by gdorsi** [comment]
If you want to try the PR before the merge this is the pages link: https://gdorsi.github.io/BangleApps/

File: index.js (hunk 1 above)
Comment (gfwilliams) [reply index.js]: I don't know... Maybe if it was an accidental disconnection it makes sense to keep it?

**Comment by gfwilliams** [comment]
This looks great - thanks!

No worries if you don't feel like it, but actually as there's a connect button, you could replace it with a 'Disconnect' one when the page is connected to the watch :)

File: index.js (hunk 1 above)
Comment (gdorsi) [reply index.js]: Ok, also the implementation may rise the complexity bar of the script

**Pull request merged**